activerecord/test/cases/strict_loading_test.rb

```diff
@@ -2,17 +2,18 @@
 
 require "cases/helper"
 require "models/developer"
+require "models/company"
 require "models/computer"
 require "models/mentor"
 require "models/project"
 require "models/ship"
+require "models/ship_part"
 require "models/strict_zine"
 require "models/interest"
+require "models/treasure"
```

Comment:
Not sure but I think `Treasure` model is not used anywhere. Maybe we can skip it.

Reply:
It's used below due to this definition on ShipPart `has_many :trinkets, class_name: "Treasure", as: :looter` 🙂 
```ruby
    assert_raises ActiveRecord::StrictLoadingViolationError do
      developer.ship.parts.first.trinkets.to_a
    end
```